client/src/components/Search/SearchBox/styles.ts

```diff
@@ -43,24 +43,12 @@ export default makeStyles((theme: Theme) => ({
       textAlign: "center",
     },
 
-    [theme.breakpoints.up("xs")]: {
+     [theme.breakpoints.down(769)]: {
```

Comment:
@duokobia,  Initial styling is set to mobile-first, using `[theme.breakpoints.down(769)]` means you have desktop-first in mind. Is that the case?

Also, can values come from them instead so that scaling in the future would be in one file? You had coded the screen size `down(769)]:***`.

Reply:
@ Maxwell-ihiaso, The mobile first approach intended did not reflect in the app's responsiveness at mobile viewports despite those lines of code. 

That informed my refactoring this way.

Values coming from the theme is a minor task as noted by @EngrZaks in our last meeting. 

A compromise based on app's design should be reached on what component(s) view(s) will be sacrificed before setting breakpoints in theme.

This is the team's decision to make.

Reply:
@duokobia, Breakpoints were set based on different screen sizes, rather than compromising on the app designs would it not be better for it to be built to meet the requirements?

It is true that values coming from the theme is a minor task but should would you advise hard coding things we can avoid and hope for a later refactoring, especially considering that you are making a change to something that already extract value from theme? You may want to agree with me that we can avoid some of the refactoring needs while implementing the code.

You mentioned that the mobile-first approach did not reflect. **Kindly see this demo on just the search component, let me know where the anomaly is.**

https://user-images.githubusercontent.com/64949174/201025656-f4773676-12d6-4608-a8c7-c9596e819b8e.mp4

In addition, it will not be so managing to have `[theme.breakpoints.up()]` and `[theme.breakpoints.down()]` used to manage the responsiveness of the same component.

As a gentle note, this refers to the search box alone. I see you refactored the serachResult. Good work!. .👏

Reply:
@duokobia, Breakpoints were set based on different screen sizes, rather than compromising on the app designs would it not be better for it to be built to meet the requirements?

It is true that values coming from the theme is a minor task but should would you advise hard coding things we can avoid and hope for a later refactoring, especially considering that you are making a change to something that already extract value from theme? You may want to agree with me that we can avoid some of the refactoring needs while implementing the code.

You mentioned that the mobile-first approach did not reflect. **Kindly see this demo on just the search component, let me know where the anomaly is.**

https://user-images.githubusercontent.com/64949174/201025656-f4773676-12d6-4608-a8c7-c9596e819b8e.mp4

In addition, it will not be so managing to have `[theme.breakpoints.up()]` and `[theme.breakpoints.down()]` used to manage the responsiveness of the same component.

As a gentle note, this refers to the search box alone. I see you refactored the serachResult. Good work!. .👏